docs/reference/aggregations/metrics/scripted-metric-aggregation.asciidoc

```diff
@@ -9,25 +9,23 @@ Example:
 
 [source,js]
 --------------------------------------------------
-POST ledger/_search?size=0
+POST /transactions/_search
 {
-    "query" : {
-        "match_all" : {}
-    },
-    "aggs": {
-        "profit": {
-            "scripted_metric": {
-                "init_script" : "params._agg.transactions = []",
-                "map_script" : "params._agg.transactions.add(doc.type.value == 'sale' ? doc.amount.value : -1 * doc.amount.value)", <1>
-                "combine_script" : "double profit = 0; for (t in params._agg.transactions) { profit += t } return profit",
-                "reduce_script" : "double profit = 0; for (a in params._aggs) { profit += a } return profit"
-            }
-        }
+  "size": 0,
+  "aggs": {
+    "profit": {
+      "scripted_metric": {
+        "init_script" : "params._agg.transactions = []",
+        "map_script" : "params._agg.transactions.add(doc['type.keyword'].value == 'sale' ? doc.amount.value : -1 * doc.amount.value)", <1>
+        "combine_script" : "double profit = 0; for (t in params._agg.transactions) { profit += t } return profit",
+        "reduce_script" : "double profit = 0; for (a in params._aggs) { profit += a } return profit"
+      }
     }
+  }
 }
 --------------------------------------------------
 // CONSOLE
-// TEST[setup:ledger]
+// TEST[setup:transactions]
```

Comment:
The name of the setup script is `ledger` so this change broke the build. If you'd like to rename it you can change it in `docs/build.gradle` so long as you change everything that uses it.